### PR TITLE
Niara

### PR DESCRIPTION
**Pull request by niajohnson26**
Added Zainab's image. I'm not sure if I did this correctly?

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -222,4 +222,8 @@ input[type="checkbox"] {
 h1 {
     font-family: "Handwriting", serif;
     font-size: 3rem;
+}
+.about {
+  color: black;
+  font
```
Comment (HenRMJ) [inline assets/css/style.css]: Missing the full property and value on line 228

File: about.html [hunk 2]
```diff
@@ -44,24 +44,35 @@ <h3>An extension of your brain</h3>
...
-        <img class="Imagegroup2" id="ZainabIbrahim" href=""/>
+        <div>
+          <a>Niara Johnson</a>
+          <a>Thanh Pham</a>
+          <a>Zainab Ibrahim</a>  
+        </div>
+        <div>
+          <img class="Imagegroup2" id="NiaraJohnson" src="./assets/images/NiaraJohnson.png" alt="Image of Niara Johnson" />
+          <img class="Imagegroup2" id="ThanhPham" src="" alt="Image of Thanh Pham" />
+          <img class="Imagegroup2" id="ZainabIbrahim" src="" alt="Image of Zainab Ibrahim" />
```
Comment (HenRMJ) [inline about.html]: Didn't add Zainab's picture in the src

**Review by HenRMJ — CHANGES_REQUESTED** [review]
Just a couple of things need to be changed!

File: assets/css/style.css (hunk 1 above)
Comment (niajohnson26) [reply assets/css/style.css]: I'm not sure who wrote this?

**Pull request closed without merging**